migrations/261_assessments__help_link__add.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE assessments ADD COLUMN help_link text;
```

Comment:
We generally do our best to make migrations idempotent, though in practice they should never run more than once:

```suggestion
ALTER TABLE assessments ADD COLUMN IF NOT EXISTS help_link text;
```

Also note that as of https://github.com/PrairieLearn/PrairieLearn/pull/6129, migration filenames must start with a timestamp prefix. See https://github.com/PrairieLearn/PrairieLearn/blob/master/migrations/README.md for more details.

Reply:
Ahh good to know. I'll go ahead and commit this fix then go back and update the filenames.

Reply:
Renaming the file(s) according to the README gives an `error: Error initializing PrairieLearn server: value "20220818192226" is out of range for type integer` when attempting to start up the server locally, am I doing something incorrect?

Reply:
You'll need to get this branch up to date with master before that will work.

Reply:
Oh right, thanks!